std/src/main/scala/algebra/std/byte.scala

```diff
@@ -0,0 +1,58 @@
+package algebra
+package std
+
+import algebra.lattice._
+import algebra.number._
+import algebra.ring._
+import algebra.std.util.StaticMethods
+
+package object byte extends ByteInstances
+
+trait ByteInstances {
+  implicit val byteAlgebra = new ByteAlgebra
+
+  val ByteMinMaxLattice = new Lattice[Byte] {
```

Comment:
I usually prefer anonymous trait style too, but some reasons for using classes could be to allow people to extend the type class implementation or allow class annotations (eg @SerialVersionUID).